portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentlibrarystore/DocumentLibraryStore.testcase

```diff
@@ -21,19 +21,11 @@ definition {
 		else {
 			User.firstLoginPG();
 
-			DMDocument.tearDownCP();
```

Comment:
the tests that create a DM without creating a new site for it, are removing it in the testcase instead of the teardown. I deleted this macro because it tries to search the DM name but the search bar is unavailable, then it shows a error message in the logs.

Reply:
Okay, I'll not remove it.